### PR TITLE
asm/ascii: pass ABI to functions

### PR DESCRIPTION
**Pull request by achille-roussel**

### CODE REVIEW

File: build/internal/x86/cpu.go [hunk 1]
```diff
@@ -13,20 +13,24 @@ import (
...
-	jump(LabelRef(jmp), f, true)
+	jump(cpuAddr, LabelRef(jmp), f, true)
 }
 
 // cpuAddr is a Mem operand containing the global symbolic reference to the
 // X86 cpu feature flags.
 var cpuAddr = NewDataAddr(Symbol{Name: "github·com∕segmentio∕asm∕cpu·X86"}, 0)
 
-func jump(jmp Op, f cpu.X86Feature, invert bool) {
+func jump(abi, jmp Op, f cpu.X86Feature, invert bool) {
```
Comment (chriso) [inline build/internal/x86/cpu.go:33]: Do we need to pass `abi` to `jumpSingleFlag`+`jumpMultiFlag` and use it in place of `cpuAddr`?

Comment (achille-roussel) [reply build/internal/x86/cpu.go:33]: Yes, if this solves the linking issues.

I just wanted to get something quick to test it out before we undertake to migrate all the function signatures.

**Comment by chenpengcheng** [comment]
@achille-roussel when do you think you can merge this?

**Comment by achille-roussel** [comment]
Not as-is, if this is the pattern we have to adopt we need to apply it to all functions who need access to the ABI

**Comment by chenpengcheng** [comment]
@achille-roussel i am blocked on this. when you get a chance to ship this, LMK.

**Pull request closed without merging**